apps/oozie/src/oozie/models.py

```diff
@@ -1721,6 +1721,8 @@ def cron_frequency(self, cron_frequency):
     data_['cron_frequency'] = cron_frequency
     self.data = json.dumps(data_)
 
+  class Meta(object):
```

Comment:
We are good I think, as those don't have a custom manager anyway. Also models.py is not used anymore, it is models2.py.

When we use a custom manager
`objects =`